src/videojs-contrib-hls.js

```diff
@@ -14,6 +14,7 @@ import {MediaSource, URL} from 'videojs-contrib-media-sources';
 import m3u8 from './m3u8';
 import videojs from 'video.js';
 import MasterPlaylistController from './master-playlist-controller';
+import Stats from './stats';
```

Comment:
I believe this will import the same object every time so that it is shared between videos. Meaning that the stats will increment forever even when we change sources.